bin/test-cleanup

```diff
@@ -12,31 +12,31 @@ fi
 
 echo "cleaning up namespace [${linkerd_namespace}] in k8s-context [${k8s_context}] and associated test namespaces"
 
-if ! namespaces=$(kubectl --context=$k8s_context get ns -oname | grep -E "/$linkerd_namespace(-|$)"); then
+if ! namespaces=$(kubectl --context=$k8s_context get ns -l linkerd.io/shared-component -oname); then
```

Comment:
I think this integration test clean-up script can be simplified further, in another PR.